internal/store/root/mount.go

```diff
@@ -213,17 +213,11 @@ func (r *Store) checkMounts() error {
 // CleanMountAlias removes all leading and trailing slashes from a mount alias.
 // Note: Slashes inside the alias are valid and will be kept.
 func CleanMountAlias(alias string) string {
-	for {
-		if !strings.HasPrefix(alias, "/") && !strings.HasPrefix(alias, "\\") {
-			break
-		}
+	for strings.HasPrefix(alias, "/") || strings.HasPrefix(alias, "\\") {
 		alias = strings.TrimPrefix(strings.TrimSuffix(alias, "/"), "/")
 		alias = strings.TrimPrefix(strings.TrimSuffix(alias, "\\"), "\\")
```

Comment:
```suggestion
		alias = strings.Trim(alias, "/")
		alias = strings.Trim(alias, "\\")
```

Reply:
We only want to remove leading and trailing slashes. strings.Trim does also remove them within the string.

Reply:
Its documentation says otherwise:
> Trim returns a slice of the string s with all leading and trailing Unicode code points contained in cutset removed

Reply:
Oh. Maybe I was wrong 🫤